docs/articles/roadmap/README.md

```diff
@@ -15,28 +16,30 @@ Please vote on the features you want us to implement first to help us prioritize
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **In Progress**
-[API Testing](https://github.com/DevExpress/testcafe/issues/4288)                                 | Planned
+[HTTP API Testing](https://github.com/DevExpress/testcafe/issues/4288)                            | Planned
+[HTTP2/HTTP3 Support](https://github.com/DevExpress/testcafe-hammerhead/issues/1960)              | Planned
+[Set HTTP Cookies](https://github.com/DevExpress/testcafe/issues/4063)                            | Planned
```

Comment:
What about `Interactive demo center`? Don't we want to include it? @kirovboris

Reply:
Yes, we were going to include.

Reply:
[The new issue](https://github.com/DevExpress/testcafe/issues/5966)